src/Provider/TemplaterServiceProvider.php

```diff
@@ -4,21 +4,14 @@
 
 use Illuminate\Support\ServiceProvider;
 
-class TemplaterServiceProvider extends ServiceProvider {
-
-    /**
-     * Indicates if loading of the provider is deferred.
-     *
-     * @var bool
-     */
-    protected $defer = false;
-
+class TemplaterServiceProvider extends ServiceProvider
+{
     /**
      * Register the service provider.
      *
      * @return void
      */
-    public function register()
+    public function boot()
```

Comment:
Just curious if this is the suggested update?

Reply:
Wasn't explicitly said in the upgrade docs. But was running into a blade compiler issue & found this answer in a forum.

https://laracasts.com/discuss/channels/general-discussion/difference-between-boot-and-register-method